broker/core/src/bbdo/stream.cc

```diff
@@ -1457,7 +1458,7 @@ void stream::acknowledge_events(uint32_t events) {
  */
 void stream::send_event_acknowledgement() {
   if (!_coarse) {
-    if (std::get<0>(_bbdo_version) >= 3) {
+    if (std::get<0>(_bbdo_version) >= 3 && std::get<0>(_bbdo_version) >= 1) {
```

Comment:
I think this is a bug. Not now but in a near future when for example the version will be 3.1.0